.github/workflows/unit_tests.yaml

```
@@ -0,0 +1,54 @@
+name: Run unit tests
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: hammer-cli-foreman-puppet
+    strategy:
+      fail-fast: false
+      matrix:
+        #ruby: [2.3, 2.4, 2.5]
+        ruby: [2.5]
```

We run on Ruby 2.5 and 2.7 now. I'd also prefer the multi line approach since it's easier to maintain IMHO.
```suggestion
        ruby:
          - 2.5
          - 2.7
```